### PR TITLE
Fix gslib struct declaration

### PR DESCRIPTION
**Pull request by kmittal2**
| PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#2418](https://github.com/mfem/mfem/pull/2418) | @kmittal2 | @tzanio | @jandrej + @brendankeith + @tzanio | 07/21/21 | 07/21/21 | 07/22/21 | |

### CODE REVIEW

**Comment by tzanio** [comment]
This PR is now [under review](https://github.com/mfem/mfem/blob/master/CONTRIBUTING.md#pull-requests) (see the table in the PR description). To help with the review process, please **do not force push** to the branch.

**Comment by tzanio** [comment]
Can we review this quickly?

**Review by jandrej — APPROVED** [review]
Already looked at it this morning. Changes look good.

**Review by kanye-quest — APPROVED** [review]
LGTM

**Comment by tzanio** [comment]
@brendankeith, can you test this + glvis/master?

**Review by brendankeith — APPROVED** [review]

**Comment by tzanio** [comment]
LGTM

File: miniapps/gslib/schwarz_ex1p.cpp [hunk 1]
```diff
@@ -327,7 +327,7 @@ int main(int argc, char *argv[])
       {
          bnd(i+d*number_boundary) = vxyz(idx + d*number_true);
       }
-      n      colorv[i] = (unsigned int)color;
+      colorv[i] = (unsigned int)color;
```
Comment (tzanio) [inline miniapps/gslib/schwarz_ex1p.cpp:330]: 😁

File: fem/gslib.cpp [hunk 2]
```diff
@@ -19,7 +19,10 @@
 #pragma GCC diagnostic ignored "-Wunused-function"
 #endif
 
+namespace gslib
+{
```
Comment (tzanio) [inline fem/gslib.cpp:23]: The `namespace` here shouldn't be necessary, can you confirm?

**Review by tzanio — APPROVED** [review]

**Comment by tzanio** [comment]
Merged in `next` for testing...

**Pull request merged**